civitai-downloader-v2/tests/api/test_websocket.py

```
@@ -0,0 +1,312 @@
+            
+            # Wait for initial progress message
+            time.sleep(0.1)
+            
+            # Skip any progress messages
+            try:
+                # Try to consume progress messages
+                while True:
+                    msg = websocket.receive_json()
+                    if msg["type"] != "download_progress":
+                        break
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)